src/Employer/Employer.Web/ViewModels/DisplayVacancyViewModel.cs

```diff
@@ -0,0 +1,100 @@
+        public string ExpectedDuration { get; internal set; }
+        public string HoursPerWeek { get; internal set; }
+        public Address Location { get; internal set; }
+        public IEnumerable<string> EmployerAddressElements { get; internal set; }
+        public string MapUrl { get; internal set; }
+        public string NumberOfPositions { get; internal set; }
+        public string NumberOfPositionsCaption { get; internal set; }
+        public string OutcomeDescription { get; internal set; }
+        public string PossibleStartDate { get; internal set; }
+        public string ProviderName { get; internal set; }
+        public IEnumerable<string> Qualifications { get; internal set; }
+        public string ShortDescription { get; internal set; }
+        public IEnumerable<string> Skills { get; internal set; }
+        public string ThingsToConsider { get; internal set; }
+        public string Title { get; internal set; }
+        public string TrainingDescription { get; internal set; }
+        public string TrainingTitle { get; internal set; }
+        public string TrainingType { get; internal set; }
+        public string TrainingLevel { get; internal set; }
+        public string VacancyDescription { get; internal set; }
+        public string VacancyReferenceNumber { get; internal set; }
+        public string WageInfo { get; internal set; }
+        public string WageText { get; internal set; }
+        public string WorkingWeekDescription { get; internal set; }
+
+        public bool HasClosingDate => !string.IsNullOrWhiteSpace(ClosingDate);
+
+        public bool HasShortDescription => !string.IsNullOrWhiteSpace(ShortDescription);
+
+        public bool HasProviderName => !string.IsNullOrWhiteSpace(ProviderName);
+
+        public bool HasTrainingTitle => !string.IsNullOrWhiteSpace(TrainingTitle);
+
+        public bool HasWorkingWeekDescription => !string.IsNullOrWhiteSpace(WorkingWeekDescription);
+
+        public bool HasHoursPerWeek => !string.IsNullOrWhiteSpace(HoursPerWeek);
+
+        public bool HasWageText => !string.IsNullOrWhiteSpace(WageText);
+
+        public bool HasWageInfo => !string.IsNullOrWhiteSpace(WageInfo);
+
+        public bool HasExpectedDuration => !string.IsNullOrWhiteSpace(ExpectedDuration);
+
+        public bool HasPossibleStartDate => !string.IsNullOrWhiteSpace(PossibleStartDate);
+
+        public bool HasTrainingLevel => !string.IsNullOrWhiteSpace(TrainingLevel);
+
+        public bool HasNumberOfPositionsCaption => !string.IsNullOrWhiteSpace(NumberOfPositionsCaption);
+        public bool HasMapUrl => !string.IsNullOrEmpty(MapUrl);
+
+        public bool HasVacancyDescription => !string.IsNullOrWhiteSpace(VacancyDescription);
+
+        public bool HasTrainingDescription => !string.IsNullOrWhiteSpace(TrainingDescription);
+
+        public bool HasOutcomeDescription => !string.IsNullOrWhiteSpace(OutcomeDescription);
+
+        public bool HasSkills => Skills == null || Skills.Any();
```

Comment:
Might be a typo here. If `Skills` is `null` then `HasSkills` will be assigned true.

Reply:
👍 Good catch